src/org/apache/xerces/dom/NodeImpl.java

```diff
@@ -151,6 +152,8 @@ public abstract class NodeImpl
 
     // data
 
+    private DOMLocator locator;
```

Comment:
can this be made final?

Reply:
Turns out it can't. There's a cloneNode method that needs to assign it.